docker/prod-server/etebase-server.ini

```diff
@@ -0,0 +1,27 @@
+[global]
+secret_file = secret.txt
+debug = false
+static_root = ./data/static
+media_root = ./data/media
+static_url = /static/
+media_url = /user-media/
+redis_uri = redis://redis:6379
+api_secret = add_api_super_secret_here
+sign_up_verification_base_domain = https://httpstat.us
+AWS_ACCESS_KEY_ID =
+AWS_SECRET_ACCESS_KEY =
+AWS_STORAGE_BUCKET_NAME = 
+; TODO update above AWS key values with prod values.
+
+; TODO: update allowed_host1 value, for prod. update with domain name.
+[allowed_hosts]
+allowed_host1 = *
+
+; TODO: update following values with prod db credentials.
+[database]
+ENGINE = django.db.backends.postgresql_psycopg2
+NAME = eteserverdb
+USER = root
+PASSWORD = mysecretpassword
```

Comment:
@vitaly512 these database values should pull from env variables, not be hardcoded.

Reply:
Okay, I will add .env file for this.